test/specs/IE11.js

```diff
@@ -1,29 +1,18 @@
-import {describe, it} from '../utils/mochaw'
+import { describe, it } from '../utils/mochaw'
+import { goToPassportUploadScreen, uploadFileAndClickConfirmButton } from './scenarios/sharedFlows.js'
 
 const options = {
-  pageObjects: ['DocumentSelector', 'Welcome', 'DocumentUpload']
+  screens: ['Confirm', 'DocumentSelector', 'Welcome', 'DocumentUpload']
```

Comment:
Should the `pageobjects` folder also be renamed `screens` for consistency?

Reply:
Yes, it should. I will change it.